save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/s3/DefaultS3Operations.kt

```diff
@@ -210,11 +196,14 @@ class DefaultS3Operations(
             .build()
     }
 
-    private fun <T : Any> CompletableFuture<T>.toMonoAndPublishOn(): Mono<T> = toMono().publishOn(scheduler)
-
     companion object {
-        private fun <T : Any> Mono<T>.handleNoSuchKeyException(): Mono<T> = onErrorResume(NoSuchKeyException::class.java) {
-            Mono.empty()
+        private val region = Region.AWS_ISO_GLOBAL
```

Comment:
some detalization maybe, like `defaultS3Region` or smth like this

Reply:
added